advanced/startup.m

```diff
@@ -0,0 +1,8 @@
+function [] = startup()
```

Comment:
I like these changes to keep "include" in a common place.  Is there still as startup in the root directory that adds include as well, or just in the subdirectories?

Reply:
there is a include/include.m that does all the imports, and a startup.m script in every sub-folder and kit. There is no startup.m in the root folder as there are no examples there.